pre_commit/clientlib.py

```diff
@@ -133,7 +135,7 @@ def remove_default(self, dct):
     *[
         cfgv.OptionalNoDefault(item.key, item.check_fn)
         for item in MANIFEST_HOOK_DICT.items
-        if item.key != 'id'
+        if item.key not in('id', 'alias')
     ]
```

Comment:
shouldn't need any changes to `CONFIG_HOOK_DICT`

Reply:
My `OptionalAlias` won't a `key` not a `check_fn`, that's why I left the `key not in ()`